conf/views/chargeA/chargeDetails.njk

```diff
@@ -0,0 +1,87 @@
+{% extends "includes/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "govuk/components/date-input/macro.njk"    import govukDateInput %}
+{% from "govuk/components/button/macro.njk"        import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "macros/title.njk"                         import title %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% block pageTitle %}
+  {{ title(messages("chargeA.chargeDetails.title"), form.errors) }}
+{% endblock %}
+
+{% block mainContent %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% if form.errors.length > 0 %}
+          {{ govukErrorSummary({
+            "titleText": messages("error.summary.title"),
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <h1 class="govuk-heading-xl">{{messages('chargeA.chargeDetails.heading')}}</h1>
+
+        <form action="{{ submitUrl }}" method="post">
+
+          {{ csrf() | safe }}
+
+          {{ govukInput({
+            id: "numberOfMembers",
+            label: {
+              text: messages('chargeA.chargeDetails.numberOfMembers.title'),
+              isPageHeading: false,
+              classes: "govuk-form-label--s"
+            },
+            name: "numberOfMembers",
+            type: "text",
+            value: form.numberOfMembers.value,
+            classes: "govuk-input--width-10",
+            errorMessage: form.numberOfMembers.error
+          }) }}
+
+        {{ govukInput({
+          id: "totalAmtOfTaxDueAtLowerRate",
+          label: {
+            text: messages('chargeA.chargeDetails.totalAmtOfTaxDueAtLowerRate.label'),
+            isPageHeading: false,
+            classes: "govuk-form-label--s"
+          },
+          name: "totalAmtOfTaxDueAtLowerRate",
+          type: "text",
+          value: form.totalAmtOfTaxDueAtLowerRate.value,
+          classes: "govuk-input--width-10",
+          errorMessage: form.totalAmtOfTaxDueAtLowerRate.error
+        }) }}
+
+         {{ govukInput({
+            id: "totalAmtOfTaxDueAtHigherRate",
+            label: {
+              text: messages('chargeA.chargeDetails.totalAmtOfTaxDueAtHigherRate.label'),
+              isPageHeading: false,
+              classes: "govuk-form-label--s"
+            },
+            name: "totalAmtOfTaxDueAtHigherRate",
+            type: "text",
+            value: form.totalAmtOfTaxDueAtHigherRate.value,
+            classes: "govuk-input--width-10",
+            errorMessage: form.totalAmtOfTaxDueAtHigherRate.error
+          }) }}
+
+
+          {{ govukButton({
+            attribute: {id: "submit"},
+            text: messages("site.continue")
+          }) }}
+
+        </form>
+
+      </div>
+    </div>
+
+   <a href= "{{ viewModel.returnUrl }}" class="govuk-link"> {{ messages("return.to.link", viewModel.schemeName) }}</a>
```

Comment:
`href= "{{ viewModel.returnUrl }}"` needs to be `href="{{ viewModel.returnUrl }}"`